app.js

```diff
@@ -1,474 +1,570 @@
-        openDropdown()
-      }
-      else {
-        itemClick($('.quickClone').attr('data-repo'))
+    $(".quickClone").click(() => {
+      if ($(".quickClone").attr("data-repo") === undefined) {
+        openDropdown();
+      } else {
+        itemClick($(".quickClone").attr("data-repo"));
       }
-    })
+    });
 
-    $('.cloneAndClose').click(() => {
-      closeModal()
-      getGithubIssue($('.cloneAndClose').attr('data-repo'), true)
-    })
+    $(".cloneAndClose").click(() => {
+      closeModal();
+      getGithubIssue($(".cloneAndClose").attr("data-repo"), true);
+    });
 
-    $('.cloneAndKeepOpen').click(() => {
-      closeModal()
-      getGithubIssue($('.cloneAndKeepOpen').attr('data-repo'), false)
-    })
+    $(".cloneAndKeepOpen").click(() => {
+      closeModal();
+      getGithubIssue($(".cloneAndKeepOpen").attr("data-repo"), false);
+    });
 
-    $('.close').click(() => {
-      closeModal()
-    })
+    $(".close").click(() => {
+      closeModal();
+    });
 
-    $('.noClone').click(() => {
-      closeModal()
-    })
+    $(".noClone").click(() => {
+      closeModal();
+    });
   }
 }
 
 function saveAppliedFilters(urlObj) {
   // check for the appropriate url
   // url should have /issues and should not track any url that has an issue number at the end
-  if (urlObj.url.indexOf('/issues') > 0 && isNaN(urlObj.issueNumber)) {
+  if (urlObj.url.indexOf("/issues") > 0 && isNaN(urlObj.issueNumber)) {
     // save the filter querystring for when/if we navigate back
-    var url = urlObj.url
-    var querystring = url.substring(url.indexOf('/issues'))
+    var url = urlObj.url;
+    var querystring = url.substring(url.indexOf("/issues"));
 
     // filter object stores the querystring, the organization and the repo
     var filter = {
       filter: querystring,
       organization: urlObj.organization,
       currentRepo: urlObj.currentRepo
-    }
-
-    chrome.storage.sync.get({
-      filters: []
-    }, (item) => {
-
-      var exists = false;
-      var changed = false;
-
-      // convert the string to an empty array for existing users
-      if (typeof item.filters === 'string') {
-        item.filters = []
-      }
-
-      item.filters.forEach((f) => {
-        // if the storage array contains the org and repo, then set exists flag
-        if (f.organization === filter.organization && f.currentRepo === filter.currentRepo) {
-          exists = true
+    };
+
+    chrome.storage.sync.get(
+      {
+        filters: []
+      },
+      item => {
+        var exists = false;
+        var changed = false;
+
+        // convert the string to an empty array for existing users
+        if (typeof item.filters === "string") {
+          item.filters = [];
+        }
 
-          // if the querystring value has changed, set the changed flag and update the filter
-          if (f.filter !== filter.filter) {
-            changed = true
-            f.filter = filter.filter
+        item.filters.forEach(f => {
```

Comment:
I'd like to keep the parenthesis around function params in arrow functions. Actually, if you can revert all code styling changes and just make the template literal changes, that would be preferable. I can make another issue to add linting/prettier and specify the rules I want. Then you can make a PR for that as well if you'd like